File: app/src/main/java/com/tanish2k09/sce/utils/TopCommentStore.java
package com.tanish2k09.sce.utils;

public class TopCommentStore {
    private static StringBuilder topComment = new StringBuilder();

    public static void appendLine(String line) {
        topComment.append(line).append('\n');
    }

    public static String getComment() {
        return topComment.toString();
    }

    public static void clear() {
        topComment = new StringBuilder();
    }
}

File: app/src/main/java/com/tanish2k09/sce/data/TopCommentStore.kt
package com.tanish2k09.sce.data

object TopCommentStore {
    private var topComment = StringBuilder()

    val comment: String
        get() = topComment.toString()

    fun appendLine(line: String) {
        topComment.append(line).append('\n')
    }

    fun clear() {
        topComment = StringBuilder()
    }
}
